core/controllers/home.py

```diff
@@ -286,20 +273,12 @@ class UploadExploration(base.BaseHandler):
     @base.require_fully_signed_up
     def post(self):
         """Handles POST requests."""
-        title = self.payload.get('title')
-        category = self.payload.get('category')
         yaml_content = self.request.get('yaml_file')
 
-        if not title:
-            raise self.InvalidInputException('No title supplied.')
-        if not category:
-            raise self.InvalidInputException('No category chosen.')
-
         new_exploration_id = exp_services.get_new_exploration_id()
         if ALLOW_YAML_FILE_UPLOAD.value:
             exp_services.save_new_exploration_from_yaml_and_assets(
-                self.user_id, yaml_content, title, category,
-                new_exploration_id, [])
+                self.user_id, yaml_content, '', '', new_exploration_id, [])
```

Comment:
Ditto here. Maybe better to default title and category in ``save_new_exploration_from_yaml_and_assets``